runners/spark/src/main/java/org/apache/beam/runners/spark/SparkPipelineOptions.java

```diff
@@ -90,8 +90,8 @@ public String create(PipelineOptions options) {
 
   @Description("Enable/disable sending aggregator values to Spark's metric sinks")
   @Default.Boolean(true)
-  Boolean getEnableSparkSinks();
-  void setEnableSparkSinks(Boolean enableSparkSinks);
+  Boolean getEnableSparkMetricSinks();
```

Comment:
IMHO, `setIsEnableSparkMetricsSink` sounds a bit off, since one's intention is to set the `enableSparkMetricsSink` to true, rather than the `isEnableSparkMetricsSink` per-se,
which is simply a utility method to query the state of `enableSparkMetricsSink`.

What do you think?